### ISSUE TITLE
Moe's Mansion extension?

### ISSUE THREAD

**Issue by Eoin-ONeill-Yokai** [flathub/org.srb2.SRB2Kart#30]
Would it be possible to add a source build of `Moe's Mansion` as an extension to the existing flatpak repo?

I know it's not an officially supported build, but it might be nice as an opt-in for users who might want faster framerates or the other additional features.

**Comment by Lonsfor** [comment]
i dont think it would be accepted as flathub only does official builds. but i have a moe mansion flatpak available [here](https://github.com/Lonsfor/org.srb2.SRB2Kart)
